python/lsst/sconsUtils/installation.py

```diff
@@ -307,7 +307,7 @@ def __call__(self, target, source, env):
         if not os.path.isdir(destpath):
             state.log.info(f"Creating directory {destpath}")
             os.makedirs(destpath)
-        for root, dirnames, filenames in os.walk(source[0].path):
+        for root, dirnames, filenames in os.walk(source[0].path, followlinks=True):
```

Comment:
@ktlim do you have an opinion on this? It fixes the problem and *seems* like it should be safe given that we shouldn't be putting symlinks into our git repos that point to `/` or something.  But we aren't enforcing that a symlink must stay within the package being built...

Reply:
> @ktlim do you have an opinion on this? It fixes the problem and _seems_ like it should be safe given that we shouldn't be putting symlinks into our git repos that point to `/` or something. But we aren't enforcing that a symlink must stay within the package being built...

The other alternative here is that I can move the YAML files into the Python source tree, but I would consider that a fairly disruptive change.

Reply:
The alternative is to reverse the symlink so that the files are really in the python tree but the symlink is from the top level. You already have a `cp` in your eups install script that would Just Work in that scenario.

Reply:
Um, can't you customize the eupspkg.cfg.sh installation for this package to build the symlink?

Reply:
Ha! Yes. Writing some bespoke copy code just for that part of the python tree would also fix it.